src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  people: Person[],
+};
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { slugId } = useParams();
+
+  const getParentLink = (name: string) => {
+    const parent = people.find(par => par.name === name);
+
+    if (parent) {
+      return (
+        <PersonLink
+          person={parent}
+        />
+      );
+    }
+
+    return name;
+  };
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {people.map(person => {
+          const {
+            sex,
+            born,
+            died,
+            fatherName,
+            motherName,
+          } = person;
+
+          const motherContent = motherName
+            ? getParentLink(motherName)
+            : '-';
+
+          const fatherContent = fatherName
+            ? getParentLink(fatherName)
+            : '-';
```

Comment:
we can avoid this two similar ternary and decide whatr render in getParentLink.
just add check on start of getParentLink util
```
if (!name) {
 return '-';
}
```
and call in jsx getParentLink
```
<td>{getParentLink(motherName)}</td>
<td>{getParentLink(fatherName)}</td>
```